README.md

```diff
@@ -434,6 +436,7 @@ The `Error Set` shows a unique set of errors returned by all issued requests. Th
     "99th": 3530000,
     "max": 3660505
   },
+  "buckets": {"0":9952,"1000000":40,"2000000":6,"3000000":0,"4000000":0,"5000000":2},
```

Comment:
This seems under-specified from a user documentation perspective. Do you think that making the keys the actual range that the bucket contains would reduce the need for additional documentation?

Reply:
There's two (mutually unrelated) things I see that could be changed, and I don't think we should do either:

1. Use the exact timespec given to vegeta, e.g. "1s" instead of "1000000".
* Makes it harder to parse the output in a unified way. The current code forces a uniform output format, no matter the input. So it's easy to write a script to analyze arbitrary vegeta JSON outputs. Parsing the time specs completely correctly is non-trivial (for non-Go code).

2. Use "${low}-${high}" as key, instead of "${low}".
* The input format wouldn't match the output format.
* Harder for downstream scripts to calculate dict keys.
* You'd need a special case for the overflow bin (e.g. "${low-}infty"). 
* It seems somewhat redundant, esp. when read manually (given the already sorted output).
* Allows corrupt values like "200-100", which is at least a hint of a bad design.
* Downstream scripts will probably generally need to sort the keys (most languages dict's are not ordered). With "${low}" that's easier (map to int, sort), with "${low}-${high}" there's an extra string parsing step.

Documentation-wise, since we print the bins in order, and there's generally a "0" bin, I think it's somewhat obvious what they mean from the example.

Caveat: The docs don't mention that the numbers are nanosec. Somewhat of an oversight. Maybe I should fix this?

Let me know if I should update the code, docs or both. I'm leaning towards only extending the docs with basically "the numbers are nanosecond lower bounds of the bins".

Reply:
Docs update sounds good to me.

Reply:
Done.